app/build.gradle

```diff
@@ -492,11 +492,10 @@ if (findProject(':wavesdk')) {
 }
 
 dependencies {
-    // To see what the latest geckoview-nightly version is go here:
-    // https://maven.mozilla.org/?prefix=maven2/org/mozilla/geckoview/geckoview-nightly-armeabi-v7a/
-    armImplementation deps.gecko_view.nightly_armv7a
-    arm64Implementation deps.gecko_view.nightly_arm64
-    x86_64Implementation deps.gecko_view.nightly_x86_64
+    // "nightly" or "beta" or "release"
+    def branch = "nightly"
```

Comment:
I wonder if branch should be set in `versions.gradle`? Probably fine since we need to think about how we are going to handle this for future releases.